pkg/apiserver/middlewares/v1/jwt.go

```diff
@@ -142,17 +143,32 @@ func Unauthorized(c *gin.Context, code int, message string) {
 
 func NewJWT(dbClient *database.Client) (*JWT, error) {
 	// Get secret from environment variable "SECRET"
-	secret := os.Getenv("SECRET")
-	if secret == "" {
-		secret = "crowdsecret"
+	var (
+		secret []byte
+	)
+
+	secret_string := os.Getenv("SECRET")
```

Comment:
Can we change this VAR name to something like `CS_LAPI_SECRET` or `CROWDSEC_LAPI_SECRET`, to avoid having same var names if crowdsec is running with other services using this var name.